src/gloss/data/primitives.clj

```diff
@@ -61,6 +61,38 @@
   [x]
   (.longValue (bigint x)))
 
+(defn bytes-umedium
```

Comment:
For consistency, these fns should have arrows in their name, a la `bytes->umedium`